ansible/playbooks/service/grafana.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: Manage Grafana
+  hosts: [ 'debops_service_grafana', 'debops_grafana' ]
+  become: True
+
+  environment: '{{ inventory__environment | d({})
+                   | combine(inventory__group_environment | d({}))
+                   | combine(inventory__host_environment  | d({})) }}'
+
+  roles:
+
+    - role: etc_services
+      tags: [ 'role::etc_services', 'skip::etc_services' ]
+      etc_services__dependent_list:
+        - '{{ grafana__etc_services__dependent_list }}'
+
+    - role: keyring
+      tags: [ 'role::keyring', 'skip::keyring', 'role::grafana' ]
+      keyring__dependent_apt_keys:
+        - '{{ grafana__keyring__dependent_apt_keys }}'
+
+    - role: ferm
+      tags: [ 'role::ferm', 'skip::ferm' ]
+      ferm__dependent_rules:
+        - '{{ grafana__ferm__dependent_rules }}'
+        - '{{ nginx__ferm__dependent_rules if (grafana__nginx_enabled|bool) else [] }}'
+
+    - role: nginx
```

Comment:
The `nginx` role has the `python` role as a dependency (I think it changed some time after you made the PR). Can you update the playbook and add it as well?

Reply:
added python dep